content/en/containers/kubernetes/distributions.md

```diff
@@ -94,7 +94,7 @@ spec:
 
 ## Azure Kubernetes Service (AKS) {#AKS}
 
-AKS requires specific configuration for the `Kubelet` integration due to AKS certificates setup.
+AKS requires a specific configuration for the `Kubelet` integration due to how AKS has setup the SSL Certificates. Additionally the optional [Admission Controller][3] feature requires a specific configuration to prevent an error when reconciling the webhook.
```

Comment:
```suggestion
AKS requires a specific configuration for the `Kubelet` integration due to how AKS has setup the SSL Certificates. Additionally, the optional [Admission Controller][3] feature requires a specific configuration to prevent an error when reconciling the webhook.
```